packages/legend-query-builder/src/components/result/QueryBuilderResultPanel.tsx

```diff
@@ -533,7 +512,11 @@ export const QueryBuilderResultPanel = observer(
       .map((item) => (
         <MenuContentItem
           key={item.key}
-          title={item.title ?? ''}
+          title={
+            !item.disableFunc?.(queryBuilderState)
```

Comment:
we shouldn't mentioned `Saved Query` within the `legend-query-builder` module as we don't know about it. To keep the modulizarity lets just do `DataCube Not Supported`